lib/ProductOpener/Ingredients.pm

```diff
@@ -5134,7 +5207,7 @@ sub develop_ingredients_categories_and_types ($ingredients_lc, $text) {
 
 		foreach my $categories_and_types_ref (@{$ingredients_categories_and_types{$ingredients_lc}}) {
```

Comment:
Do we need to define an **$alternate_names_regexp** there as well?

There is only German for now, maybe German accents (for oil, for example) should be handled there?

Reply:
The alternate names are only for finding the right name associated with a type, for matching the categories we still use the categories field. Maybe I'm misunderstanding what you want to do, could you give me an example of something that you want to add parsing support for, and what it should be parsed to?

Reply:
Ah, I understand now. If there is no need of regex after, then we don't need that. Okay okay.


But what if we have ingredient:
Pflanzliche ole (Kurbiskern)

(Instead of "Pflanzliche öle (Kürbiskern)" with accents)

Category: Pflanzliche ole (ok)
Type: Kurbiskern (ok)
alternate_names: Kurbiskernöl (not ok but not related to alternate_names)

So yes, you are totally right, we don't need $alternate_names_regexp

Kurbiskern vs kürbiskern would be handled in canonicalize_taxonomy_tag, is that correct? 
If yes, then, all good you can close that comment.